crates/meilisearch/tests/documents/geojson/mod.rs

```diff
@@ -418,3 +418,36 @@ async fn geo_radius() {
     }
     "#);
 }
+
+#[actix_rt::test]
```

Comment:
this should not be part of this PR?

Reply:
Yes, that's why we're waiting for #5907 to land before merging this one 😁 
Since I updated the Cargo.lock on both PR they'll probably have a conflict, so I based this one on #5907